### ISSUE TITLE
didEndDraggingOverlay is called after a moveOverlay call

### ISSUE THREAD

**Issue by joustava** [applidium/ADOverlayContainer#21]
This is not a bug and intentional I'd guess, but what do you think about separating 'dragging' and calling moveOverlay callbacks, e.g something similar to
```
func overlayContainerViewController(_ containerViewController: OverlayContainerViewController,
                                        didEndDraggingOverlay overlayViewController: UIViewController,
                                        transitionCoordinator: OverlayContainerTransitionCoordinator) {}
```
The above would be the original callback but only called when a user actually drags.
```
func overlayContainerViewController(_ containerViewController: OverlayContainerViewController,
                                        didEndMovingOverlay overlayViewController: UIViewController,
                                        transitionCoordinator: OverlayContainerTransitionCoordinator) {}
```
The above would be an additional callback but only called after moveOverlay call has finished?

Thoughts?

**Comment by gaetanzanella** [comment]
Hi @joustava, could you provide an use case of your proposal ?

**Comment by joustava** [comment]
I don't have a real use-case at the moment. I just noticed that `didEndDraggingOverlay` is called for when a user actually drags the overlay **and also** when I'd just trigger the `moveOverlay` from code.  This last confused me a little as I wasn't expecting `didEndDraggingOverlay` to be called when a user doesn't drag and the `moveOverlay` would be called to e.g dismiss the overlay after tapping a button.

Might be I am slightly abusing this library, and what I am actually looking for is to check when the overlay is not shown at all, so that I can replace it's content when that happens.

**Comment by gaetanzanella** [comment]
I'm working on OverlayContainer 2.0. Showing/hiding should be easier, thanks to the method `canReachNotchAt`. Would you like to try it ?

**Comment by gaetanzanella** [comment]
https://github.com/applidium/ADOverlayContainer/tree/feature/overlayContainerV2

**Comment by gaetanzanella** [comment]
I released 2.0.0, try it directly 🙂

**Comment by joustava** [comment]
Just managed to update to 2.0. Works like a charm. Thanks!
